.rubocop.yml

```diff
@@ -148,7 +184,7 @@ Style/IfUnlessModifier:
   Enabled: true
 
 # Checks the indentation of the first element in an array literal.
-Layout/FirstArrayElementIndentation:
+Layout/IndentFirstArrayElement:
```

Comment:
I think these names actually upset rubocop because they have since changed.

Reply:
Oh whooops I didn't realize there was already a rubocop in this repo...gonna undo that